docs/api/_src/pipeline_lint_tests/multiqc_config.md

```diff
@@ -0,0 +1,4 @@
+multiqc_config
```

Comment:
I think that you need to delete this and re-render it using the python script in the api docs folder. I merged in a couple of PRs changing how this works yesterday.

Reply:
How do I see if it worked? I am not sure if I understand.

Reply:
I'll take a look locally..

Reply:
See 89886fd4, all I did was this:

```console
$ cd docs/api/
$ rm _src/pipeline_lint_tests/multiqc_config.md
$ python make_lint_md.py
multiqc_config
```

Due to https://github.com/nf-core/tools/pull/1245 being merged yesterday.